asapdiscovery/data/utils.py

```diff
@@ -670,17 +788,61 @@ def filter_molecules_dataframe(
             # Keep pIC50 string
             # Use default pIC50 error
             print(row)
-            pIC50, pIC50_stderr = pIC50, "0.5"  # strings
+            pIC50 = pIC50
+            ## Set as high number so sorting works but still puts this at end
+            pIC50_stderr = 100
+            pIC50_lower = row[f"{assay_name}: IC50 CI (Lower) (µM)"]
+            pIC50_upper = row[f"{assay_name}: IC50 CI (Upper) (µM)"]
 
         pIC50_series.append(pIC50)
-        pIC50_stderr_series.append(pIC50_stderr)
+        pIC50_stderr_series.append(float(pIC50_stderr))
+        pIC50_lower_series.append(pIC50_lower)
+        pIC50_upper_series.append(pIC50_upper)
 
     mol_df["pIC50"] = pIC50_series
     mol_df["pIC50_stderr"] = pIC50_stderr_series
+    mol_df["pIC50_95ci_lower"] = pIC50_lower_series
+    mol_df["pIC50_95ci_upper"] = pIC50_upper_series
+
+    ## Compute binding affinity in kcal/mol
+    # use kT = 0.593, kcal/mol for 298 K/25C
+    deltaG = lambda pIC50: -0.593 * np.log(10.0) * float(pIC50)
```

Comment:
We should do a better job of the conversion here:
* Have the `temperature` explicitly specified here so we know what assay temperature is assumed. (We could even ask Haim what temperature their facility is controlled to.)
* Use the Km for the peptide substrate to convert to a real DeltaG using [Cheng-Prusoff](https://pharmacologycanada.org/Cheng-Prusoff-equation). From [slack](https://covidmoonshot.slack.com/archives/C01J3BBP6CA/p1671451633342139?thread_ts=1671016079.148199&cid=C01J3BBP6CA), we have the Km ~ 9.5 µM for the peptide used here. From Matteo Ferla (who emailed Haim):
> The Vmax was not converted into kcat, but left in fluorescence (2,000 AU/min). It's machine dependent but given that ~10,000 AU is saturation in most machines (AFAIK) and it's 375 nM ([5-FAM]-AVLQ↓SGFR-[Lys(Dabcyl)]-K-amide substrate, it strikes me as a reassuringly healthy enzyme. (I say this because in [Okamoto et al 2010](https://pubmed.ncbi.nlm.nih.gov/21087086/) for SARS-original Mpro the numbers are baffling: a hydrolase with a kcat of 0.007 s-1 for its physiological activity is really bad)